sdk/core/Azure.Core/src/DelayStrategy.cs

```diff
@@ -45,19 +43,19 @@ protected DelayStrategy(TimeSpan? maxDelay = default, double jitterFactor = Defa
             TimeSpan? initialDelay = default,
             TimeSpan? maxDelay = default)
         {
-            return new ExponentialDelayStrategy(initialDelay ?? TimeSpan.FromSeconds(0.8), maxDelay ?? TimeSpan.FromMinutes(1));
+            initialDelay ??= RetryOptions.DefaultInitialDelay;
+            maxDelay ??= RetryOptions.DefaultMaxDelay;
```

Comment:
nit - should we put these defaults on the parameters themselves - this might make the defaults more transparent.

Reply:
Interesting -- I can't think of a reason not to -- let me look more closely.

Reply:
Actually, looks like this would be a breaking change:

> **D.11.18 Changing the Default Value for an Optional Parameter**
> Optional parameters are a compile-time feature in which any caller that did not provide a value for a parameter gets the value from the target member’s metadata. Changing the default value has no effect on callers until they recompile, which can create confusion where the compiled binary and the developer’s understanding of the program from the source code do not match.
> 
> The .NET BCL team considers this an unacceptable breaking change due to the possibility of developer confusion.